.travis/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [ "$TYPE" = 'style' ]; then
+	sudo apt-get -yqq update
+	sudo apt-get install shellcheck
+else
+
```

Comment:
Minor, but there seem to be some additional newlines in this `if [ ]` block as well as the the `else` block in the `install.sh` file.